### DISCUSSION TITLE
Web site not fully loaded

### DISCUSSION THREAD

**Discussion by krik0v** [snipe/snipe-it#14409] [Q&A]
Hi,

I have installed the latest Snipe-IT on AWS EC2 (Linux Ubuntu) 
At the beginning, the site was fully loaded with all the graphics. Now it looks like that.

![image](https://github.com/snipe/snipe-it/assets/114931967/455e2ba6-5e2b-4069-a001-542c446aa2c6)

Is there any Service  I need to check that is running? 

Itzik.

**Comment by ksgrg** [comment]
What is the web server you are using? may be your host configuration isn't done properly.
Check the developer tools to see what errors you are seeing on the browser.
